Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/exile/BanishmentInfluenceTest.java

```diff
@@ -0,0 +1,62 @@
+package org.mage.test.cards.abilities.oneshot.exile;
```

Comment:
Should this go under the CLB folder or in here? @theelk801

Reply:
yeah, that’s probably a better place to put it

Reply:
where?

Reply:
> where?

instead of `cards/abilities/...` /cards/single/clb`. I don't think it currently exists so you'll have to make the folder.